demos-europe/demosplan-core
# feat(refs T33536): tweak import menu entry in sidemenu

https://yaits.demos-deutschland.de/T33536


- The menu entry for import should be renamed to `Import(Stellungnahmen)`
- The menu entry should be in the analysis section and not in the configuration section anymore

![Screenshot 2023-11-15 at 14-47-44 Übersicht EIN TEST Raumordnung Online](https://github.com/demos-europe/demosplan-core/assets/91727189/46b12f44-b637-4748-9443-c12b395eb20b)

### How to review/test

- checkout branch
- build either `blp`or the `regio` project
- login as FPA user
- confirm the entries of the side menu are in the correct spot


### PR Checklist

related PR to override the translation for ewm: https://github.com/demos-europe/demosplan-project-ewm/pull/97

Delete the checkbox if it doesn't apply/isn't necessary.
